src/main/java/com/tripply/booking/model/request/AmentiesRequest.java

```diff
@@ -11,10 +11,14 @@
 @NoArgsConstructor
 public class AmentiesRequest {
 
-    @NotBlank(message = "amenityName can't be empty")
```

Comment:
Please provide correct message here "Amenity can't be left blank"

Reply:
I haven't seen this type of message in the whole development career you provide this message. I think my message is right. Jatin, please let me know if any changes is required.

Reply:
Please check my comment again